adding_service_cluster/adding-service.adoc

```diff
@@ -1,23 +1,31 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/attributes-openshift-dedicated.adoc[]
+
 [id="adding-service"]
 = Adding services to a cluster using {cluster-manager-first} console
+
 :context: adding-service
 
 toc::[]
 
 You can add, access, and remove add-on services for your {product-title}
+
```

Comment:
@rolfedh Yeah and not around all conditionals. We really want the includes only. 
@jhradilek Can you clarify where we need a blank line in cases where we have something like:

`ifdef::openshift-rosa[]`
`include::modules/adding-service-existing.adoc[leveloffset=+1]`
`endif::openshift-rosa[]`
`include::modules/access-service.adoc[leveloffset=+1]`

Do we need to end up at:
`ifdef::openshift-rosa[]`
`include::modules/adding-service-existing.adoc[leveloffset=+1]`
`endif::openshift-rosa[]`
`//line space here`
`include::modules/access-service.adoc[leveloffset=+1]`